Documentation/disaster-recovery.md

```diff
@@ -47,10 +47,31 @@ In the event of partial or total self-hosted control plane loss, `bootkube
 recover` may be able to assist in re-bootstrapping the self-hosted control
 plane.
 
-The `bootkube recover` subcommand does not recover a cluster directly. Instead,
-it extracts the control plane configuration from an available source and
-renders manifests in a format that `bootkube start` can use invoked to reboot
-the cluster.
+The `bootkube recover` subcommand does not recover a cluster directly. The
+recovery is a two step process: `bootkube recover` and `bootkube start`. The
+recovery extracts the control plane configuration from an available source and
+renders manifests to the local filesystem in a format that `bootkube start` can
+be invoked to reboot the cluster.
+
+There are two available sources to choose from: etcd or an API server.
+
+### What does bootkube recover do?
+
+Bootkube attempts to read the configuration from an existing backend etcd or API
+server. On success, the tool renders a modified bootstrap control plane to
+support direct injection into a kubelet.
```

Comment:
Clarify what "direct injection into a kubelet" means. I assume it means, roughly, "in a format that `bootkube start` can provide to the kubelet, as static pods, to restart the control plane bootstrap process".

Reply:
Or maybe more simply "in a format that `bootkube start` can handle", since the user doesn't really "directly inject" static pods, but rather just calls `bootkube start` (which is documented elsewhere).

Reply:
good call.. @xiang90 @colemickens I believe the current commits address your comments.